src/main/java/org/homeschoolpebt/app/submission/messages/ConfirmationMessage.java

```diff
@@ -22,12 +22,31 @@ public ConfirmationMessage(Submission submission, Transmission transmission) {
 
   @Override
   public Email renderEmail() {
-    var applicationNumber = SubmissionUtilities.getFormattedConfirmationNumber(transmission.getConfirmationNumber());
-    String subject = "Test confirmation email";
+    var confirmationNumber = SubmissionUtilities.getFormattedConfirmationNumber(transmission.getConfirmationNumber());
+    var applicantFullName = SubmissionUtilities.applicantFullName(submission);
+    String subject = "Application Submitted for P-EBT 4.0";
+    String body = """
+      <html>
+        <body>
+          <p>Dear %s</p>
```

Comment:
It's not super duper important, but is this %s HTML-injectable? It's probably smart to escape this and/or use a Thymeleaf template so we get auto-escaping.

Reply:
It's usually best to do "Dear %s," i.e. have a comma in the "Dear..." line. Not a blocker.

Reply:
Not injectable as far as I can tell. `formatted` builds the string right there to pass to the mailgun client.

I agree with the comma, but I went with what the copy said. I felt it was too minor to call out